src/views/admin-results-listing.phtml

```diff
@@ -44,6 +44,9 @@ require_once('partials/head.phtml');
                     <td><?php echo $result['name'] ?> </td>
                     <td><?php echo $result['quiz_template_id'] ?></td>
                     <td>We don't believe that numbers represent the value of people.</td>
+                    <td class="text-center">
+                        <a class="btn btn-primary py-1" href="/result/<?php echo $result['user_id']; ?>/<?php echo  $result['quiz_instance_id']; ?>">View</a>
```

Comment:
An URI helper would be an improvement here, and code would look cleaner as well.

Reply:
I already have an open issue regarding this improvement and it will be treated separately if it's ok with you :) @live-happy-code-hard